src/components/menus/main-menu/main-menu--children.twig

```diff
@@ -0,0 +1,51 @@
+{% if menu_type == 'main' %}
+<div class="menu-container {{ menu_bg }}">
+  <nav role="navigation" aria-labelledby="-menu" class="nav  main--secondary--horizontal block block-menu navigation menu--main">
```

Comment:
Can we remove any classes from this that are not UIDS-specific? This looks like it was output copied from a Drupal site.

Reply:
yeah we can. I had just copied this over for testing purposes so we can strip it down.